src/fragment-components/a-icons.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import { Search, Dropdown } from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { css } from 'emotion';
+import { ComponentInfo } from '.';
+import * as Icons from '@carbon/icons-react';
+import image from './../assets/component-icons/icons.svg';
+import { ElementTile } from '../components/element-tile';
+import _ from 'lodash';
```

Comment:
Why not import a specific function you use instead? Like the rest of the codebase does.